bin/unix/install-dependencies/opensuse.sh

```diff
@@ -0,0 +1,39 @@
+#/bin/bash
+#
+## 
+## Usage: build-opensuse
+## 
+## This script automates the building of ownCloud documentation on the openSUSE platform.
+## While new, it should handle installing all of the required platform depencies, and afterwards
+## build the documentation. 
+##
+## Author: Matthew Setter <msetter@owncloud.com>
+##
+
+function install_dependencies()
+{
+    # first check if the repository's already been added
+    zypper lr | grep devel_languages_python &>/dev/null
+
+    if [ "$?" -eq 1 ]; then
```

Comment:
Grep has a silent mode just for this kind of thing that runs a lot faster than piping it's output to /dev/null. I suggest replacing this if statement with:

    if zypper lr |  grep -q devel_languages_python; then